core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages.properties

```diff
@@ -285,6 +285,11 @@ FlightRecordingSupportRule_EA_TEXT_WARN_LONG=This recording is from an early acc
 FlightRecordingSupportRule_NO_JVM_VERSION_EVENTS_TEXT=There were no VM information events with JVM version information recorded. Please turn on JVM information events to use this rule.
 FlightRecordingSupportRule_UNSUPPORTED_TEXT_WARN_SHORT=The recording is from an unsupported JRE version.
 FlightRecordingSupportRule_UNSUPPORTED_TEXT_WARN_LONG=This recording is from a runtime with JRE version {jdkVersion}, before JDK Flight Recorder was fully supported in HotSpot. Versions before 7u40 only have partial support for flight recordings. The automated analysis is not supported, and you may see errors when attempting to analyze this recording.
+JfrPeriodicEventsFixRule_RULE_NAME=JFR Periodic Events Fix
+JfrPeriodicEventsFixRule_TEXT_OK=The version of OpenJDK you are running is not affected by a perfomance issue related to periodic events.
+JfrPeriodicEventsFixRule_NO_JVM_VERSION_EVENTS_TEXT=There were no VM information events with JVM version information recorded. Please turn on JVM information events to use this rule.
+JfrPeriodicEventsFixRule_UNSUPPORTED_TEXT_WARN_SHORT=The version of OpenJDK 8 you are running is lacking an important performance fix.
+JfrPeriodicEventsFixRule_UNSUPPORTED_TEXT_WARN_LONG=The version of OpenJDK 8 you are running is lacking an important performance fix. Upgrading to version 8u302 or later can, in some cases, improve your performance quite radically. See JDK-8266723 for more information. An alternative can be to disable any periodic events not needed, such as the NativeExecutionSample event.
```

Comment:
Is it possible to embed http link ?

It would be nice to be able to click directly on the bug id [JDK-8266723](https://bugs.openjdk.java.net/browse/JDK-8266723) to get more details.

Reply:
I naively tried `<a href=""></a>`, which did not work. Unfortunately I am not familiar with `ResourceBundle`s.